app/controllers/supplejack_api/supplejack_application_controller.rb

```diff
@@ -27,7 +27,11 @@ def authenticate_user!
 
       if current_user
         if current_user.over_limit?
-          error_message = I18n.t('users.reached_limit')
+          error_message = if RecordSchema.roles[current_user.role.to_sym].try(:anonymous)
```

Comment:
Should we add a test? There was a 500 being returned when Dan showed us.

Reply:
Yeah that was because the anonymous user didn't have an email address associated with it.

Reply:
We should probably make the email a required field on an API user